src/audio/Audio.js

```diff
@@ -143,7 +143,7 @@ class Audio extends Object3D {
 
 				// ensure _progress does not exceed duration with looped audios
 
-				this._progress = this._progress % ( this.duration || this.buffer.duration );
+				this._progress = this._progress % ( this.duration || ( this.buffer ? this.buffer.duration : Number.MAX_VALUE ) );
```

Comment:
This code section is only executed when the audio is playing. So just creating an audio and calling `pause()` does not cause any issues. Your application seems to call `audio.play()`  but without a valid audio buffer which is wrong in the first place. And tbh I don't understand how the suggested fix is supposed to help users.

Reply:
If anything, I would suggest to throw an error if `buffer` is not set at this point.